src/main/webapp/consentGroupContainer/ConsentGroupContainer.js

```
@@ -51,20 +58,24 @@ export const ConsentGroupContainer = hh(class ConsentGroupContainer extends Comp
   // history
   getHistory() {
     ProjectMigration.getHistory(this.props.consentKey).then(resp => {
-      this.setState(prev => {
-        prev.history = resp.data;
-        return prev;
-      })
-    });
+      if (this._isMounted) {
+        this.setState(prev => {
+          prev.history = resp.data;
+          return prev;
+        });
+      }
+    }).catch(() => {});
```

avoid empty catch-all. Throw error or log it.